src/app/containers/Caption/index.test.jsx

```diff
@@ -6,27 +6,29 @@ import { blockContainingText } from '../../models/blocks';
 
 const newsServiceContextStub = {
   imageCaptionOffscreenText: 'Image caption, ',
+  videoCaptionOffscreenText: 'Video caption, ',
 };
 const persianServiceContextStub = {
   imageCaptionOffscreenText: ' ، عنوان تصویر',
+  videoCaptionOffscreenText: ' ، عنوان ویدئو',
 };
 
 const block = blockContainingText('caption', 'Some caption text...');
 
-const CaptionWithContext = contextStub => (
+const CaptionWithContext = (contextStub, type) => (
   <ServiceContext.Provider value={contextStub}>
-    <Caption block={block} />
+    <Caption block={block} type={type} />
   </ServiceContext.Provider>
 );
 
```

Comment:
We should add enzyme tests to assert that the output of the Caption has the correct offscreen text. For example: `<span>Image caption, </span>` for the image case.  Here's an example of another container that has assertion-based unit tests: https://github.com/bbc/simorgh/blob/latest/src/app/containers/Headings/index.test.jsx#L152

Reply:
@sareh Are you saying we should have an assertion based enzyme test and a different kind of test just checking for the type, or just the former?

Reply:
We can use enzyme to check the various cases for the type. For example, rendering `CaptionWithContext` with persian context and type `video` should output the `persianServiceContextStub.videoCaptionOffscreenText` value in the offscreen span. If we pass in the type `image` it should have the value `persianServiceContextStub.imageCaptionOffscreenText`, etc.